context-core/pom.xml

```diff
@@ -64,30 +64,24 @@
 
         <!-- Jena dependencies -->
         <dependency>
-            <groupId>com.hp.hpl.jena</groupId>
-            <artifactId>arq</artifactId>
-            <version>2.8.7</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>log4j</groupId>
-                    <artifactId>log4j</artifactId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.apache.jena</groupId>
+            <artifactId>jena-arq</artifactId>
+            <version>3.17.0</version>
         </dependency>
         <dependency>
-            <groupId>com.hp.hpl.jena</groupId>
-            <artifactId>tdb</artifactId>
-            <version>0.8.10</version>
+            <groupId>org.apache.jena</groupId>
+            <artifactId>jena-tdb</artifactId>
+            <version>4.3.2</version>
```

Comment:
The plan is to remove `sdb` and solely use `tdb`, which will cleanup the versions.

But this will be done in a separate PR to keep this PR manageable.